honestcash-v2/src/app/editor/components/header/header.component.html

```diff
@@ -1,7 +1,7 @@
 <header>
   <nav class="navbar navbar-light navbar-expand">
     <a class="navbar-brand" href="/">
-      <app-honest-logo></app-honest-logo>
+      <core-honest-logo></core-honest-logo>
```

Comment:
these core components are shared via sharedModule (inside /core folder) because they are used in multiple feature modules